docs/user-guide/configure-xmem-server.md

```diff
@@ -139,14 +127,54 @@ Because the ZIS server makes z/OS security calls it restrits which clients are a
+
+- JCL member `ZWESASTC` is copied from `SZWESAMP` installation PDS to a PDS on the JES concatenation path. 
+- The PDSE load library `SZWEAUTH`is APF-authorized, or load module `ZWESAUX` is copied to an existing APF Auth LoadLib.
+- The load module `ZWESAUX` must run in key 4 and be non-swappable by adding a PPT entry to the SCHEDxx member of the system PARMLIB `PPT PGMNAME(ZWESAUX) KEY(4) NOSWAP`.
+
+#### Zowe Auxiliary Address space
+
+The cross memory server runs as a started task `ZWESISTC` that uses the load module `ZWESIS01`.
+
+In some use cases, the Zowe cross memory server has to spawn child address spaces, which are known as auxiliary (AUX) address spaces.  The auxiliary address spaces run as the started task `ZWESASTC` using the load module `ZWESAUX` and are started, controlled, and stopped by the cross memory server.  
+
+An example of when an auxiliary address space is used is for a system service that requires supervisor state but cannot run in cross-memory mode. The service can be run in an AUX address space which is invoked by the Cross Memory Server acting as a proxy for unauthorized users of the service. 
+
+Do not install the Zowe auxiliary address space unless a Zowe extension product's installation guide explicitly asks for it to be done.  This will occur if the extension product requires services of Zowe that cannot be performed by the cross memory server and an auxiliary address space needs to be started.  
+
+A default installation of Zowe does not require auxiliary address spaces to be configured.
+
+:::info**Important**
+The cross memory `ZWESISTC` task starts and stops the `ZWESASTC` task as needed. **Do not start the `ZWESASTC` task manually.**
+:::
+
 ## Summary of cross memory server installation
 
 You can start the cross memory server using the command `/S ZWESISTC` once the following steps have been completed.
 
-- JCL member `ZWESLSTC` is copied from `SZWESAMP` installation PDS to a PDS on the JES concatenation path.
-- The PDSE Load Library `SZWEAUTH`is APF-authorized, or Load module `ZWESI00` is copied to an existing APF Auth LoadLib.
+- JCL members `ZWESLSTC` and `ZWESASTC` are copied from `SZWESAMP` installation PDS to a PDS on the JES concatenation path.
+- The PDSE Load Library `SZWEAUTH` is APF-authorized, or Load modules `ZWESI00` and `ZWESAUX` are copied to an existing APF Auth LoadLib.
 - The JCL member `ZWESLSTC` DD statements are updated to point to the location of `ZWESI00` and `ZWESIP00`. 
-- The load module `ZWESI00` must run in key 4 and be non-swappable by adding a PPT entry to the SCHEDxx member of the system PARMLIB `PPT PGMNAME(ZWESI00) KEY(4) NOSWAP`.
+- The load modules `ZWESI00` and `ZWESAUX` must run in key 4 and be non-swappable by adding a PPT entry to the SCHEDxx member of the system PARMLIB 
+```
+PPT PGMNAME(ZWESI00) KEY(4) NOSWAP
+PPT PGMNAME(ZWESAUX) KEY(4) NOSWAP
+```
```

Comment:
Missed this during the first review, there is no load module named ZWESI00, it should be ZWESIS01. I've briefly searched and it seems that there was an issue for that before: https://github.com/zowe/docs-site/issues/2635 
But it is still exist here and migrated to zos-components-installation-checklist